hieradata_aws/production.yaml

```diff
@@ -19,7 +19,7 @@ govuk::apps::content_publisher::google_tag_manager_id: "GTM-NQXC4TG"
 govuk::apps::content_publisher::google_tag_manager_preview: "env-7"
 govuk::apps::content_store::performance_platform_big_screen_view_url: 'https://performance-platform-big-screen-view-production.cloudapps.digital'
 govuk::apps::content_store::performance_platform_spotlight_url: 'https://performance-platform-spotlight-production.cloudapps.digital'
-govuk::apps::email_alert_api::db::backend_ip_range: '10.2.3.0/24'
+govuk::apps::email_alert_api::db::backend_ip_range: '10.12.3.0/24'
```

Comment:
Is this supposed to be `10.13` instead?

Reply:
Yeah I wasn't sure. I followed the original schema as I thought there might be a good reason for it? But if nobody can think of any we might as well put it on the prod range.

Reply:
I think the production AWS hieradata was copied from staging verbatim, hence `10.2` in places that should be different. Let's put it on the production range, as it's production.